src/tests/functional/inference_engine/CMakeLists.txt

```diff
@@ -58,6 +58,10 @@ if (NOT ENABLE_OV_PADDLE_FRONTEND)
     list(APPEND EXCLUDED_SOURCE_PATHS "${CMAKE_CURRENT_SOURCE_DIR}/paddle_reader")
 endif()
 
+if (NOT ENABLE_LTO)
```

Comment:
Why do you add this test only with disabled LTO?

Reply:
Misprint from my side, thanks!

Reply:
Decided to add the test in next PR